packages/yarn-plugin-allow-scripts/README.md

```diff
@@ -1,15 +1,29 @@
 # @lavamoat/yarn-plugin-allow-scripts
 
-A Yarn v3 plugin for running `@lavamoat/allow-scripts` after install.
+A Yarn Berry plugin for integrating `@lavamoat/allow-scripts` lifecycle-script protection.
 
 ### Install
 
-Yarn v3 plugins are installed via public URL.
+#### Yarn v4
+
+First, perform basic [configuration](https://github.com/LavaMoat/LavaMoat/blob/main/packages/allow-scripts/README.md#configure) `@lavamoat/allow-scripts` for the package.
+
+The latest version of the plugin can then be installed by its URL:
 
 ```sh
 yarn plugin import https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js
 ```
 
+#### Yarn v3
+
+The current version of the plugin does not support Yarn v3. Yarn v3 users are encouraged to migrate to an actively supported package manager such as Yarn v4.
```

Comment:
Isn't Yarn v1 EOL'd?

Reply:
It's a bit ambiguous if it's actually EOL but i nany case:
- It's still downloaded and utilized by Yarn 3/4
  - https://github.com/LavaMoat/LavaMoat/pull/1218/files#diff-28322e804f60a520aed793eef6737b40eaf3f04a896a3f669c0ee4e277422d29R201
  - -> https://github.com/LavaMoat/LavaMoat/pull/1218/files#diff-6f9be4f4e3e42ee652af6f870b5c8800b967642aefdc237f66b26a3484c19ab4
  - -> https://github.com/yarnpkg/berry/issues/6258 
- How's that relevant here?

Reply:
We don't support yarn1 with the plugin but we do support configuring for it in allow-scripts. allow-scripts works fine with yarn1 and there's no reason to stop that.
I requested we keep the v3 version easily reachable and documented because it's value and we already have it. 

Overall, the goal of allow-scripts is to be viable for any repository containing a package.json